components/feature/session/build.gradle

```
@@ -21,7 +21,14 @@ android {
     }
 }
 
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
```

Let's use lazy tasks here
```suggestion
tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach() {
```